bin/deploy.js

```diff
@@ -11,7 +10,7 @@ const createStepFunction = require("../src/aws/createStepFunction");
 //   Specify files to retrieve by a workflow name
 const basenamesAndZipBuffers = getBasenamesAndZipBuffers("lambdas");
 const { lambdaRoleName, statesRoleName } = require("../src/config/roleNames");
-const stateMachineName = process.argv[2] || "example-workflow"; // TODO: perhaps throw an error?
+const stateMachineName = require("../src/util/workflowName");
 
```

Comment:
I think line 13 can be removed. I can read in the `stateMachineName` in `src/aws/generateStateMachineParams.js`